styles/components/ItemFilters.module.scss

```diff
@@ -1,30 +1,44 @@
 @import "../utils/mixins";
 
-.itemFilterOptionsContainer {
-  position: absolute;
-  word-wrap: break-word;
-  z-index: 1;
-  background-color: white;
-  width: 230px;
-  padding: 10px;
-  border: 1px solid var(--nypl-colors-ui-gray-medium);
-}
-
+// The gray box that holds the filter groups
 .filtersContainer {
+  display: flex;
+  flex-wrap: wrap;
   background-color: var(--nypl-colors-ui-gray-x-light-cool);
   padding: 16px;
   min-height: 128px;
   z-index: 0;
+  gap: var(--nypl-space-s);
 }
 
-.filterGroup {
-  display: flex;
-  width: fit-content;
+// filter group is a generic class for any filters that may exist. As of 11/22
+// there are the aggregation filters and the search by year filter
+.filterGroupMobile {
+  flex-grow: 1;
+  & .itemFilterOptionsContainer {
```

Comment:
The ampersand is not necessary here. The styles will compile to `.filterGroupMobile .itemFilterOptionsContainer`. If the usage for this is to have more specificity, you can remove the space `&.itemFilterOptionsContainer`. When compiled, the output will be `.filterGroupMobile.itemFilterOptionsContainer`.